hoprd/rest-api/src/lib.rs

```diff
@@ -257,7 +279,9 @@ async fn build_api(
                 .route("/node/entryNodes", get(node::entry_nodes))
                 .route("/node/metrics", get(node::metrics))
                 .route("/peers/:peerId/ping", post(peers::ping_peer))
-                .route("/session", post(session::create_client))
+                .route("/session/:protocol", post(session::create_client))
+                .route("/session/:protocol", get(session::list_clients))
+                .route("/session/:protocol", delete(session::close_client))
```

Comment:
Should be specific to the port.

```suggestion
                .route("/session/:protocol/:port", delete(session::close_client))
```

Reply:
Port is not enough, it needs to also know the host (interface ip) the listener is bound to.